config/database.yml

```diff
@@ -8,14 +8,14 @@ default: &default
 development:
   <<: *default
   url: postgresql://localhost/i_have_i_need_development
-  username: dev_local_user
-  password: password
+  # username: dev_local_user
```

Comment:
Hm, what's this for? Seems unrelated to this feature, no?

Reply:
lol, this is me making it work with my local postgres. undone